lib/child-writer/index.js

```diff
@@ -170,14 +182,102 @@ exports.write = (image, drive, options) => {
     }
   }
 
+  // Timeout function that only runs if we exceed the specified timeout,
```

Comment:
The wording here is a bit confusing, as it's the timeout ID that's being stored in here, not a function in that sense – can we adjust this accordingly?

Reply:
Also, is there a reason this isn't on the `state` object, or can we move that there as well?

Reply:
True – I wanted to link what the timeout actually does with its id variable when I wrote that. I'll prefix it with "Id of" to clarify.

We could move it there as well, but since `state` is mostly used for debugging I was thinking to keep the two separate.